app/src/main/java/org/oppia/android/app/help/thirdparty/LicenseListViewModel.kt

```diff
@@ -0,0 +1,36 @@
+package org.oppia.android.app.help.thirdparty
+
+import android.annotation.SuppressLint
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.android.R
+import org.oppia.android.app.viewmodel.ObservableViewModel
+import javax.inject.Inject
+
+/** View model in [LicenseListFragment]. */
+class LicenseListViewModel @Inject constructor(
+  val activity: AppCompatActivity,
+  val index: Int
+) : ObservableViewModel() {
+  private val arrayList = ArrayList<LicenseItemViewModel>()
+
+  /** Stores the list of third-party dependencies. */
+  val licenseItemList: List<LicenseItemViewModel> by lazy {
+    getRecyclerViewItemList()
+  }
+
+  @SuppressLint("ResourceType")
+  private fun getRecyclerViewItemList(): ArrayList<LicenseItemViewModel> {
+    val licenses =
+      activity.resources.obtainTypedArray(R.array.third_party_dependency_license_names_array)
+    val stringArrayResId = licenses.getResourceId(index, -1)
```

Comment:
Default value should be `0` ? Similar to `LicenseTextViewModel ` file

Reply:
Done.